### PR TITLE
Add benchmarks documentation

### PR DESCRIPTION
**Pull request by lucperkins**
Currently, all of the documentation for the OpenMessaging [benchmarks](https://github.com/openmessaging/openmessaging-benchmark/) project is scattered across READMEs in that repo. This PR moves that documentation into the OpenMessaging website.

**NOTE**: this is now ready for review

### CODE REVIEW

**Comment by sijie** [comment]
Awesome job! @lucperkins

**Comment by sijie** [comment]
@vongosling @vintagewang @zhouxinyu can you help reviewing this PR? This is improving the open messaging website.

**Comment by sijie** [comment]
Ping @vongosling @vintagewang @zhouxinyu

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.4.1
```
Comment (vongosling) [inline .ruby-version]: Do you run the website well if let ruby version to 2.4

**Comment by vongosling** [comment]
@lucperkins well done. we will merge it soon. Some minor polish will be finished, such as RocketMQ->Apache RocketMQ, some more clearly mission of OpenMessaging Benchmark Parts. Feel free to feedback when all have been done :-)

**Comment by zhouxinyu** [comment]
@lucperkins well done, let's move openmessaging-benchmark forward ~

**Comment by sijie** [comment]
@zhouxinyu is the website automatically built after merged?

**Comment by sijie** [comment]
@vongosling @zhouxinyu thank you for your reviews

**Comment by vongosling** [comment]
@lucperkins @sijie thanks for contribution, glad to see more contributors involved in openmessaging :-)

**Comment by zhouxinyu** [comment]
@sijie  yes, travis will build the `source` branch triggered by commits and then publish it.

**Pull request merged**